networks/fabric/v2/v2.1.0/2org1peergoleveldb_raft/api/fabric-api-solo-go.yaml

```diff
@@ -72,7 +72,7 @@ channels:
     - id: fixed-asset
```

Comment:
Chaincode id needs to be changed to `fixed-asset-base`
Chaincode metadataPath needs to be changed to `src/fabric/api/fixed-asset-base/metadata`